data/src/main/java/tech/pegasys/artemis/data/TimeSeriesRecord.java

```diff
@@ -19,10 +19,15 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializerProvider;
 import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonObject;
 import java.io.IOException;
```

Comment:
was it not possible to implement using jackson?  It's kind of weird to have to ways to implement json here

Reply:
Yeah...that's much simpler as well. 
Modify code with required changes. Please review.

Reply:
🎉 Excellent! Thanks!!